json-schemas/src/account-stats.json

```diff
@@ -0,0 +1,802 @@
+          "description": "Total number of refused requests (due to account limits)."
+        },
+        "apiRequests.tokenRequests.succeeded": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of token requests made."
+        },
+        "apiRequests.tokenRequests.failed": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of failed token requests."
+        },
+        "apiRequests.tokenRequests.refused": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of refused token requests (due to permissions or rate limiting)."
+        },
+        "apiRequests.push.succeeded": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of Push requests made."
+        },
+        "apiRequests.push.failed": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of failed Push requests."
+        },
+        "apiRequests.push.refused": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of refused Push requests."
+        },
+        "apiRequests.other.succeeded": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of requests made, excluding token and Push requests."
+        },
+        "apiRequests.other.failed": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of failed requests, excluding token and Push requests."
+        },
+        "apiRequests.other.refused": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of refused requests, excluding token and Push requests."
+        },
+        "push.channelMessages": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of Push channel messages."
+        },
+        "push.notifications.delivered": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of delivered Push notifications."
+        },
+        "push.notifications.failed": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of failed Push notifications."
+        },
+        "push.notifications.refused": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of refused Push notifications."
+        },
+        "push.notifications.all": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of Push notifications."
+        },
+        "push.directPublishes": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of direct publishes."
+        }
```

Comment:
Account stats has a few fields that aren't in app stats but which we probably want in here. The xchg ones I think we can ignore for now (unless/until we decide what's happening with api streamer), but we do want `peakRates` here.

so that's the following new entries:
peakRates.messages
peakRates.apiRequests
peakRates.tokenRequests
peakRates.connections
peakRates.channels
peakRates.reactor.httpEvent
peakRates.reactor.amqp
peakRates.reactor.externalQueue
peakRates.reactor.webhook
peakRates.pushRequests

Reply:
Thanks Simon - I've added these now